dbreps2/src/main.rs

```diff
@@ -1,11 +1,21 @@
 use anyhow::Result;
+use clap::Parser;
 use dbreps2::Report;
 use log::{error, info};
 use mysql_async::Pool;
 
 mod enwiki;
 mod general;
 
+/// Parsing args, yo
+#[derive(Parser, Debug)]
+#[clap(author, version, about, long_about = None)]
+struct Args {
+    /// Report name such as "enwiki:UserCats"
+    #[clap(short, long)]
+    report: String,
```

Comment:
Since this is optional, it should be typed as `Option<String>`.

Reply:
See https://github.com/clap-rs/clap/blob/v3.0.14/examples/tutorial_derive/03_02_option.rs

Reply:
Thanks, I saw this in the docs but couldn't figure out the `match.args` bit.